public/js/user.js

```diff
@@ -334,6 +335,23 @@ apos.define('apostrophe-workflow', {
       });
     };
 
+    self.enableRevert = function() {
+      apos.ui.link('apos-workflow-revert', null, function($el, id) {
+        apos.ui.globalBusy(true);
+        self.api('revert', {id: id}, function (result) {
+          apos.ui.globalBusy(false);
+          if (result.status && result.status !== 'ok') {
+            return apos.notify('Error reverting commit:' + result.status);
+          } else if (!result.status) {
+            return apos.notify('Error reverting commit');
+          }
+
+          return apos.notify('Document reverted to commit!');
+          // @@TODO - where do we go now?
```

Comment:
This is where you would redirect to `_url` if available, which should be self-evident, or notify "Document reverted to commit" if not available.